src/jobs/test-package.yml

```diff
@@ -10,17 +10,30 @@ parameters:
   node_version:
     description: version of node to run tests against
     type: string
-    default: latest
+    default: 'latest'
   upload-coverage:
     description: if true, the job will submit code coverage report to codecov.io
     default: false
     type: boolean
+  os:
+    description: operating system you want to test against
+    type: enum
+    enum: ['linux','windows']
+    default: 'linux'
 
 executor:
-  name: default
-  tag: << parameters.node_version >>
+  name: << parameters.os >>
 
 steps:
+  - run:
```

Comment:
Since we're using this logic in two places, I'd like to see it pulled out into a single command. Something like this:

`src/commands/install-node.yml`
```
# Copyright (c) 2018, salesforce.com, inc.
# All rights reserved.
# SPDX-License-Identifier: BSD-3-Clause
# For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause

# How to author commands: https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-commands
description: >
  configure github username and email

parameters:
  version:
    description: version of node to install and use
    default: latest
    type: string
steps:
  - run:
      name: Install and use node version
      command: |
          if [ <<parameters.node_version>> = "latest" ]; then
              nvm install --latest-npm
          else
              nvm install <<parameters.version>>
          fi
          nvm ls
```

And then in the job you can just add that command as one of the steps, e.g.

```
steps:
  - install-node:
       version: <<parameters.node_version>>
  - checkout
```